docs/conventions.md

```diff
@@ -49,9 +49,9 @@ The `kyt test` command finds all files with `*.test.js` in the `/src` directory.
 
 The `kyt lint` command finds all files with `.js` extension in `/src`
 
-Specify lint rules with a `.eslintrc` file in the root of your project.
+Specify lint rules with a `.eslintrc.json` file in the root of your project.
```

Comment:
What happens in the case where the user already has an `.eslintrc.json` (and also for `.styleintrc.json`)? My understanding is their file would get backed up and a new kyt-version would be created instead? It's not clear reading this that setup takes care of that for the user but changes can be made in that file after setup. Maybe that's out of scope for this specific doc, but it might be worth mentioning?